ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java

```diff
@@ -8,91 +8,41 @@
-
-    @Given("^(\\w+) opens \"([^\"]*)\" page$")
-    public void sheOpensApiClientConnectorPage(String user, String pageName) {
-        customizations.equals("open API Client Connectors page");
-        apiClientConnectorsPage.validate();
-    }
-
-    @And("^(\\w+) navigates to the next Api Connector wizard step \"([^\"]*)\"$")
-    public void sheNavigatesToTheNextApiConnectorWizardStep(String user, String step) {
-        wizard.nextStep();
-
-        //validation ensures that multiple clicking the Next button doesn't click on the same button multiple times before the next step loads and
-        // displays
-        wizard.getCurrentStep().validate();
-    }
-
-    @And("^(\\w+) creates new connector$")
-    public void sheCreatesNewConnector(String user) {
-        wizard.nextStep();
-    }
+    private static ApiClientConnectors apiClientConnectorsPage = new ApiClientConnectors();
 
     @Then("^(\\w+) opens new Api Connector wizard$")
-    public void sheOpensNewApiConnectorWizard(String user) throws Throwable {
+    public void openNewApiConnectorWizard(String user) throws Throwable {
         apiClientConnectorsPage.startWizard();
-        wizard = new ApiClientConnectorWizard();
     }
 
-    @Then("^(\\w+) uploads swagger file$")
-    public void sheUploadsSwaggerFile(String user, DataTable fileParams) throws Throwable {
-        uploadSwaggerWizardStep.validate();
-        List<List<String>> dataRows = fileParams.cells(0);
-        List<String> sourceTypes = new ArrayList<String>();
-        List<String> urls = new ArrayList<String>();
-
-        for (List<String> row : dataRows) {
-            sourceTypes.add(row.get(0));
-            urls.add(row.get(1));
-        }
-
-        uploadSwaggerWizardStep.upload(sourceTypes.get(0), urls.get(0));
-    }
-
-    @Then("(\\w+) sets up security by \"([^\"]*)\"$")
-    public void sheSetsUpSecurityBy(String user, String authType) throws Throwable {
-        securityWizardStep.validate();
-        //String authType, String authenticationUrl, String authorizationUrl, String scopes
-        switch (authType) {
-            case "OAuth 2.0":
-                securityWizardStep.setUpOAuth2Security(TestConfiguration.syndesisUrl() + TestConfiguration.syndesisCallbackUrlSuffix());
-                break;
-            default:
-                Assert.fail("The Auth type < " + authType + "> is not implemented by the test.");
-        }
+    @Then("^(\\w+) opens the API connector \"([^\"]*)\" detail$")
+    public void openApiConnectorDetail(String userName, String connectorName) {
+        apiClientConnectorsPage.clickConnectorByTitle(connectorName);
     }
 
     @Then("(\\w+) is presented with the new connector \"([^\"]*)\"$")
-    public void sheIsPresentedWithTheNewConnector(String user, String connectorName) throws Throwable {
+    public void checkNewConnectorIsPresent(String user, String connectorName) throws Throwable {
```

Comment:
I wonder since you're on refactor path. Should be `user` parameter dropped also, as there's no use for it right now.

Reply:
It depends on the readability impact as we don't have roles in the app (yet?). We can discuss users in feature-steps separately. This applies to all scenarios.

Reply:
No we don't yet. And imho we don't need to enforce role-to-account check on every step. I'd assume that once roles are present login method will take care of a user selection and we'd add a separate scenario to check if specific actions are reachable. E.g. start scenario with "login user with role xy" and all the steps performed in a sequence assume that actions are doable.

Reply:
Are we waiting for someone with merging?

Reply:
Well, I guess we aren't. As we settled discussion around user on the meeting.